src/helpers/formulaHelper.js

```diff
@@ -0,0 +1,187 @@
+const formulas = require('./formulas.json')
+
+const TOKEN_TYPE = {
+    Number: Symbol("number"),
+    Variable: Symbol("variable"),
+    Function: Symbol("function"),
+    Operator: Symbol("operator")
+};
+
+const FUNCTION_TOKENS = [
+    "MIN",
+    "MAX",
+    "ROUND"
+];
+
+function getFormulaByKey(key) {
+    return formulas[key];
+}
+
+function parse(formulaObj) {
+    const constants = formulaObj.constants;
+    let tokens = formulaObj.formula.split('');
+
+    let resultStack = [];
+    let parseStack = [];
+
+    for (let i = 0; i < tokens.length; i++) {
+        if (tokens[i] == ' ')
+            continue;
+        
+        if (tokens[i] >= '0' && tokens[i] <= '9') {
+            let number = "";
+            while (i < tokens.length && tokens[i] >= '0' && tokens[i] <= '9') {
```

Comment:
idem

Reply:
Which idem is this one? Extracting number to constants or extracting `if` to function?

Reply:
Extracting into a function

Reply:
This and other comments regarding extracting to function, I'll see what I can do